src/Panel/Model.php

```diff
@@ -332,7 +332,7 @@ public function options(array $unlock = []): array
 	{
 		$options = $this->model->permissions()->toArray();
 
-		if ($this->model->isLocked()) {
+		if ($this->lock()['isLocked'] === true) {
```

Comment:
`Question:` @bastianallgeier What do you think about having `$this->lock()` actually return the `Lock` object and not the array? It would make these occasions a lot cleaner.